www/web-proxy-sso/src/opnsense/mvc/app/models/OPNsense/ProxySSO/ProxySSO.xml

```diff
@@ -16,5 +16,8 @@
             <W2008>Windows 2008 with AES</W2008>
         </OptionValues>
     </ADKerberosImplementation>
+    <KerberosHostName type="TextField">
+        <Required>Y</Required>
```

Comment:
Needs &lt;default/&gt; set to non-zero default and model bump, also plugin version bump to 2.1